settings.gradle

```diff
@@ -1,5 +1,6 @@
 include ':sqlbrite'
 include ':sqlbrite-kotlin'
 include ':sample'
+include ':checks'
```

Comment:
nit: can you name this 'lint-checks' or 'sqlbrite-lint'? 'checks' is a bit vague.